### ISSUE TITLE
AWS ebs is always 8gb size

### ISSUE THREAD

**Issue by moosilauke18** [hashicorp/terraform#328]
When I start a new instance, it is always only 8gb which can cause a lot of problems as it isn't very efficient.  I need to then manually change the size.  Is there an easier way to change the size of the ebs disk in terraform?

**Comment by mcoms** [comment]
This works for me in `v0.3.5`, specifying...

```
block_device {
  device_name = "/dev/sda1"
  volume_type = "gp2"
  volume_size = 50
}
```

Replaces the default 8GB boot volume with a 50GB one.

**Comment by catsby** [comment]
Closing, this appears fixed by #440.
